File: 9998OS_04_FD_Code/video 4.2/SwiftInObjC/SwiftInObjC/ViewController.m
//
//  ViewController.m
//  SwiftInObjC
//
//  Created by Dan Beaulieu on 12/11/15.
//  Copyright © 2015 Dan Beaulieu. All rights reserved.
//

#import "ViewController.h"
#import "SwiftInObjC-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    HelloFromSwift *hello = [[HelloFromSwift alloc] init];
    //NSString *hwo = hello.message;
    NSString *hw = [hello getMessage];

    NSLog(@"hello %@!", hw);
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 9998OS_06_FD_Code/9998OS_06_Code/6.4/breakpoints 2/breakpoints/breakpoints/ViewController.swift
//
//  ViewController.swift
//  breakpoints
//
//  Created by Dan Beaulieu on 12/27/15.
//  Copyright © 2015 Dan Beaulieu. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        breakpointTest()
    }

    func breakpointTest() {
        var val = 0
        
        for index in 0...1000 {
            val = index
            //print("value is \(val)")
            if val == 950 {
                do {
                    try someFunction()
                } catch let ex {
                    print(ex)
                }
            }
        }
    }
    
    func someFunction() throws {
        
       buffer()
   
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

